bottlesdevs/Bottles
# Fix #1325 Desktop file does not execute exe if the bottle has a space in its name

# Description
Seems like the desktop file was generated using the bottle name and not the bottle path. These are not the same if a bottle include a space in its name

Fixes #1325

## Type of change

# How Has This Been Tested?
- Create a bottle with a space in its name
- Install a program and create and a desktop entry
- Click desktop file and observe the progam starting
